server-function/src/serveBinaris.ts

```diff
@@ -58,6 +58,7 @@ app.post('/invoke', express.json(), async (req, res) => {
 });
 
 app.use(express.static(buildDir));
+app.use((_req, res) => res.sendfile(pathResolve(buildDir, 'index.html')));
```

Comment:
Can you make sure content-type / charset match the ones served by static serving?

Reply:
Express res.sendfile seems deprecated and recommends res.sendFile instead

Internally it's implemented yhe same as static so content-type should be correct

Reply:
`Content-Type: text/html; charset=UTF-8`, it's the same.
Changing to `sendFile` and on `GET` requests.